client/dashboard/sites/monitoring-performance-card/monitoring-performance-card.tsx

```diff
@@ -0,0 +1,100 @@
+import { useMemo } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { PeriodData, useSiteMetricsQuery } from '../../../sites/monitoring/hooks/use-metrics-query';
```

Comment:
For consistency, let's rewrite the query and put it in `api-queries` package. Currently, it's using a different pattern for the query key etc.

Reply:
Thanks for taking a look.
I updated the code to reuse the existing `siteMetricsQuery` from `api-queries`. Can you please check it out and see if it's in line with the Hosting Dashboard approaches.